docs/casengines.md

```diff
@@ -1,172 +1,153 @@
 
-<img src="/docs/assets/engines.png" alt="OpenEBS storage engines - Jiva and cStor" width="900"/>
+Notes:
+- <sup>*</sup> Some flavors of Local PV support RAID/Mirroring to protect against disk failures
+- <sup>**</sup> Support for incremental snapshots/backups is not yet implemented in Mayastor 
 
-*In the above figure:*
 
-**SP** - Storage Pool, the custom resource that represents the Jiva storage pool
+## Data Engine Capabilities
 
-**CV** - cStor Volume, the custom resource that represents the cStor volume
+Data Engines are what maintain the actual state generated by the Stateful applications and are concerned about providing enough storage capacity to store the state and ensure that state remains intact over its lifetime. For instance state can be generated once, accessed over a period of next few minutes or days or modified or just left to be retrieved after many months or years. The capabilities provided by the data engines can be classified as follows:
 
-**CVR** - cStor Volume Replica
+All OpenEBS Data Engines support:
+- Dynamic Provisioning of Persistent Volumes
+- Strong Data Consistency 
 
-**SPC** - Storage Pool Claim, the custom resource that represents the cStor pool aggregate
+Below table identifies few differences among the different OpenEBS CAS engines. 
 
-**CSP** - cStor Storage Pool, the custom resource that represents cStor Pool on each node
+| Feature                                      | Jiva    |  cStor   |  Mayastor  | LocalPV  | 
+| -------------------------------------------- | :---:   | :------: | :--------: | :------: |
+| Near disk performance                        |  No     |   No     | Yes        | Yes      |
+| Full Backup and Restore using Velero         |  Yes    |   Yes    | Yes        | Yes      |
+| Synchronous replication                      |  Yes    |   Yes    | Yes        | No       |
+| Protect against node failures (replace node) |  Yes    |   Yes    | Yes        | No       |
+| Use with ephemeral storage on nodes          |  Yes    |   Yes    | Yes        | No       |
+| Thin Provisioning                            |  Yes    |   Yes    | Yes        | Yes**    |
+| Suitable for high capacity (>50GB) workloads |  No*    |   Yes    | Yes        | Yes**    |
+| On demand capacity expansion                 |  No     |   Yes    | Planned    | Yes**    |
+| Snapshots                                    |  No     |   Yes    | Planned    | Yes**    |
+| Clones                                       |  No     |   Yes    | Planned    | Yes**    |
+| Disk pool or aggregate support               |  No*    |   Yes    | Planned    | Yes**    |
+| Disk resiliency (RAID support )              |  No*    |   Yes    | Planned    | Yes**    |
+| Incremental Backups                          |  No     |   Yes    | Planned    | No**     |
 
 
+Notes:
+- <sup>*</sup> In case of Jiva data engine, the data is saved in sparse files within a mounted hostpath. This impacts the following:
```

Comment:
Why we need * in the notes? We can remove it to avoid any confusion. Its looks good without these *.

Reply:
This is a standard practice to attach additional notes. till we find a better way. Will go with this.